data/BSatishSutar.json

```
@@ -1,37 +1,95 @@
-  "type": "personal",
-  "bio": "Non-IT turned Digital Nomad. Travel enthusiast Yoga & Tech aficionado",
-  "links": [
-    {
-      "name": "LinkedIn",
-      "url": "https://www.linkedin.com/in/satish-sutar/",
-      "icon": "FaLinkedin"
-    },
-    {
-      "name": "Twitter",
-      "url": "https://twitter.com/SGSMaven",
-      "icon": "FaTwitter"
-    },
-    {
-      "name": "Medium",
-      "url": "https://medium.com/@CloudDevOpsEngineer",
-      "icon": "FaMedium"
-    },
-    {
-      "name": "GitHub",
-      "url": "https://github.com/BSatishSutar",
-      "icon": "FaGithub"
-    },
-    {
-      "name": "GitLab",
-      "url": "https://gitlab.com/SatishSutar",
-      "icon": "FaGitlab"
-    },
-    {
-      "name": "Hashnode",
-      "url": "https://hashnode.com/@Satish-S",
-      "icon": "FaHashnode"
-    }
-  ]
-}
+	"name": "Satish Sutar",
+	"type": "personal",
+	"bio": "Non-IT turned Digital Nomad. Travel enthusiast Yoga & Tech aficionado",
+	"tags": [
+		"DevOps Engineer",
+		"Cloud Engineer",
+		"Open Source",
+		"Technical Content Writer",
+		"Blogging"
+	],
+	"socials": [{
+			"icon": "FaTwitter",
+			"url": "https://twitter.com/SGSMaven"
+		},
+		{
+			"icon": "FaGithub",
+			"url": "https://github.com/BSatishSutar"
+		},
+		{
+			"icon": "SiHashnode",
+			"url": "https://hashnode.com/@Satish-S"
+		},
+		{
+			"icon": "SiMedium",
+			"url": "https://medium.com/@CloudDevOpsEngineer"
+		}
+	],
+	"links": [{
+			"name": "LinkedIn",
+			"url": "https://www.linkedin.com/in/satish-sutar/",
+			"icon": "FaLinkedin"
+		},
+		{
+			"name": "Twitter",
+			"url": "https://twitter.com/SGSMaven",
+			"icon": "FaTwitter"
+		},
+		{
+			"name": "Medium",
+			"url": "https://medium.com/@CloudDevOpsEngineer",
+			"icon": "FaMedium"
+		},
+		{
+			"name": "GitHub",
+			"url": "https://github.com/BSatishSutar",
+			"icon": "FaGithub"
+		},
+		{
+			"name": "GitLab",
+			"url": "https://gitlab.com/SatishSutar",
+			"icon": "FaGitlab"
+		},
+		{
+			"name": "Hashnode",
+			"url": "https://hashnode.com/@Satish-S",
+			"icon": "FaHashnode"
+		}
+	],
+	"milestones": [{
+			"title": "Started Freelancing",
+			"date": "Jan 2022",
+			"icon": "FaDollarSign",
+			"color": "blue",
+			"description": "Started Content Writing",
+			"url": "https://hashnode.com/@Satish-S"
+		},
+		{
+			"title": "Joined Open Source",
+			"date": "March 2023",
+			"icon": "FaGithub",
+			"color": "red",
+			"description": "",
+			"url": "https://github.com/BSatishSutar"
+		},
+		{
+			"title": "Started Tech Blog, Newsletter",
+			"date": "Dec 2022",
+			"icon": "FaLinkedin",
+			"color": "blue",
+			"description": "Blogs on Cloud and DevOps",
+			"url": "https://www.linkedin.com/newsletters/7023169120795803648/"
+		},
+		{
+			"title": "Started Independent Newsletter",
+			"date": "Dec 2022",
+			"icon": "SiBeehiiv",
+			"color": "blue",
+			"description": "Cloud and DevOps",
+			"url": "https://cloud101-newsletter-09c659.beehiiv.com/"
+		}
+	]
+
+
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

ok. Let me work on it.